validator_worker/src/core/follower_rules.rs

```diff
@@ -4,51 +4,58 @@ pub fn is_valid_transition(channel: &Channel, prev: &BalancesMap, next: &Balance
     let sum_prev: BigNum = prev.values().sum();
     let sum_next: BigNum = next.values().sum();
 
-     let deposit = channel.deposit_amount.clone();
+    let deposit = channel.deposit_amount.clone();
 
-     let prev_checks = prev.iter().all(|(acc, bal)| match next.get(acc) {
+    let prev_checks = prev.iter().all(|(acc, bal)| match next.get(acc) {
         Some(next_bal) => next_bal >= bal,
         None => false,
     });
 
-     // no need to check if there are negative balances as we don't allow them using BigUint
+    // no need to check if there are negative balances as we don't allow them using BigUint
     sum_next >= sum_prev && sum_next <= deposit && prev_checks
 }
 
- pub fn is_healthy(channel: &Channel, our: &BalancesMap, approved: &BalancesMap) -> bool {
-    // TODO: Move this to config
-    let health_threshold_neg: BigNum = (1_000 - 950).into();
+pub fn is_healthy(
+    channel: &Channel,
+    our: &BalancesMap,
+    approved: &BalancesMap,
+    health_threshold_neg: BigNum,
+) -> bool {
     let sum_our: BigNum = our.values().sum();
 
-     let zero = BigNum::from(0);
+    let zero = BigNum::from(0);
     let sum_approved_mins = our
         .iter()
         .map(|(acc, val)| val.min(approved.get(acc).unwrap_or(&zero)))
         .sum();
 
-     if sum_approved_mins >= sum_our {
+    if sum_approved_mins >= sum_our {
         return true;
     }
 
-     let deposit = &channel.deposit_amount;
+    let deposit = &channel.deposit_amount;
     let acceptable_difference = deposit * &health_threshold_neg / &BigNum::from(1_000);
```

Comment:
I think this should be 👇 where `health_threshold` is the user configuration health threshold value that is passed into the pure function
``` 
let health_threshold_neg: BigNum = (1_000 - health_threshold).into();	
```